src/components/AddWallet/AddWallet.jsx

```diff
@@ -17,93 +17,97 @@ import {
   Switch,
   useDisclosure
 } from '@chakra-ui/react';
+import i18next from 'i18next';
 
 export const AddWallet = () => {
+  const { isOpen, onClose, onOpen } = useDisclosure();
+
   const {
     register,
     handleSubmit,
-    formState: { errors, isValid }
+    formState: { errors }
```

Comment:
Do we need this property in our codebase for now?

Reply:
we an leave it it's fine